### PR TITLE
`MeasureToolbar`: modify `ArgumentException` message

### PR DESCRIPTION
**Pull request by williambohrmann3**
When registering the `DependencyProperty`s for `LineSketchEditor` and `AreaSketchEditor`, they are both using the same method for the `PropertyMetadata` `PropertyChangedCallback` parameter. This is fine, but currently the exception message when the new value is null doesn't make sense for `AreaSketchEditor`. To remedy this, I would like to remove the unused method and modify the currently used method to make sense in both cases.

Also, I removed some redundant "using" statements and corrected a typo.

### CODE REVIEW

**Comment by williambohrmann3** [comment]
Any idea why the job might fail if the solution is building for me locally?

**Comment by williambohrmann3** [comment]
All resolved - updated the certificate for UWP, changes included in this PR.

File: src/Toolkit/Toolkit.UI.Controls/MeasureToolbar/MeasureToolbar.cs [hunk 1]
```diff
@@ -597,15 +591,15 @@ private static void OnMapViewPropertyChanged(DependencyObject d, DependencyPrope
         /// Identifies the <see cref="LineSketchEditor"/> dependency property.
         /// </summary>
         public static readonly DependencyProperty LineSketchEditorProperty =
-            DependencyProperty.Register(nameof(LineSketchEditor), typeof(SketchEditor), typeof(MeasureToolbar), new PropertyMetadata(null, OnLineSketchEditorPropertyChanged));
+            DependencyProperty.Register(nameof(LineSketchEditor), typeof(SketchEditor), typeof(MeasureToolbar), new PropertyMetadata(null, OnSketchEditorPropertyChanged));
 
-        private static void OnLineSketchEditorPropertyChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        private static void OnSketchEditorPropertyChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
```
Comment (dotMorten) [inline src/Toolkit/Toolkit.UI.Controls/MeasureToolbar/MeasureToolbar.cs:596]: Why this change? It's literally the 'LineSketchEditor' property. Please undo

File: src/Toolkit/Toolkit.UI.Controls/MeasureToolbar/MeasureToolbar.cs [hunk 2]
```diff
@@ -631,26 +625,7 @@ private static void OnLineSketchEditorPropertyChanged(DependencyObject d, Depend
         /// Identifies the <see cref="AreaSketchEditor"/> dependency property.
         /// </summary>
         public static readonly DependencyProperty AreaSketchEditorProperty =
-            DependencyProperty.Register(nameof(AreaSketchEditor), typeof(SketchEditor), typeof(MeasureToolbar), new PropertyMetadata(null, OnLineSketchEditorPropertyChanged));
```
Comment (dotMorten) [inline src/Toolkit/Toolkit.UI.Controls/MeasureToolbar/MeasureToolbar.cs:634]: This is the only bug. It should call  OnAreaSketchEditorPropertyChanged instead, and don't delete the code below.

**Review by dotMorten — CHANGES_REQUESTED** [review]

**Review by dotMorten — APPROVED** [review]

**Pull request merged**